web/components/links/create-links-button.tsx

```diff
@@ -0,0 +1,192 @@
+import clsx from 'clsx'
+import { useState } from 'react'
+import { Col } from '../layout/col'
+import { Row } from '../layout/row'
+import { Title } from '../title'
+import { User } from 'common/user'
+import { ManalinkCardPreview, ManalinkInfo } from 'web/components/manalink-card'
+import { createManalink } from 'web/lib/firebase/manalinks'
+import { Modal } from 'web/components/layout/modal'
+import Textarea from 'react-expanding-textarea'
+import dayjs from 'dayjs'
+import Button, { SubmitButton } from '../button'
+import getLinkUrl from 'functions/src/get-manalink-url'
+import { DuplicateIcon } from '@heroicons/react/outline'
+
+export function CreateLinksButton(props: {
+  user: User
+
+  highlightedSlug: string
+  setHighlightedSlug: any
+}) {
+  const { user, highlightedSlug, setHighlightedSlug } = props
+
+  const [open, setOpen] = useState(false)
+
+  function updateOpen(newOpen: boolean) {
+    setOpen(newOpen)
+  }
```

Comment:
Could be an anonymous function inline for `setOpen={}`

Reply:
Done